development/development/phpstorm.rst

```diff
@@ -70,14 +70,10 @@ The default inspection settings should work just fine. However there are a coupl
 
 * If PhpStorm is not connected to your database server, you may want to turn off **SQL > SQL Dialect Detection** and **SQL > No data sources configured** warnings.
 * phpBB uses fully qualified namespaces, so you can turn off this inspection warning **PHP > Code Style > Unnecessary fully qualified name**.
-* You may enable additional JavaScript inspections. Under **JavaScript > Code quality tools**, you can enable JSCS, JSHint, JSlint and/or ESLint. phpBB comes with configuration files for JSHint and JSCS. With these inspections enabled, you can use the configuration files provided by phpBB instead of PhpStorm's default settings. To do so, point PhpStorm to the custom configuration files under **Language & Frameworks > JavaScript > Code Quality Tools**. Point JSHint to `.jshintrc` and point JSCS to `.jscsrc`.
+* You may enable additional JavaScript inspections. Under **JavaScript > Code quality tools**, you can enable ESLint. phpBB comes with configuration for XO.js which is a wrapper for ESLint. With these inspections enabled, you can use the configuration files provided by phpBB instead of PhpStorm's default settings. XO.js integration does not exsits for PhpStorm. XO.js is a wrapper tool for ESlint that is run through NodeJS on the command line. To use XO.js, make sure the NodeJS framework is downloaded and installed on your computer. Then you can install XO.js via the npm package manager and configure the ESlint options in PhpStorm. You can follow these  `Instructions <https://github.com/jamestalmage/xo-with-webstorm>`_ to get everything setup.
```

Comment:
Why is XO.js mentioned as a wrapper for ESLint twice? Too many words! The extraneous info about XO.js require NODE should still be in a separate warning as was done below previously. This paragraph should get to the point succinctly by just telling the user what to do. The extra info can be contained in the warning below requiring NodeJS should be in the warning since users may most likely already have it (see the stylelint warning below).